src/dialog/overflow-menu/overflow-menu.component.ts

```diff
@@ -78,6 +83,11 @@ export class OverflowMenu {
 
 	@Input() placement: "bottom" | "top" = "bottom";
 
+	/**
+	 * This specifies is custom trigger template was provided
+	 */
+	@Input() customTrigger = false;
```

Comment:
Maybe make this take a full template? Something like `@Inupt() customIcon: TemplateRef<any>` ... then the default icon can just be the default value to `customIcon` ... then we're also not running into potential conflicts with any content children selectors

Reply:
Alternatively we could use `ngComponentOutlet` (https://angular.io/api/common/NgComponentOutlet) and have `customIcon` just take in a component class ...

Reply:
Thanks for the review! Changing the `customTrigger` type to `TemplateRef<any>` is a great idea, however setting it's default value is not a common practice - I was searching for solutions done in this repo, and looks like `*ngIf` has to stay.